enhancements/bare-metal-fulfillment/README.md

```diff
@@ -0,0 +1,360 @@
+---
+title: bare-metal-fulfillment
+authors:
+  - Tzu-Mainn Chen
+creation-date: 2025-09-08
+last-updated: 2025-09-15
+tracking-link:
+  - None
+see-also:
+  - None
+replaces:
+  - None
+superseded-by:
+  - None
+---
+
+# Bare Metal Fulfillment
+
+## Summary
+
+One key component of Virtual Data Center (VDC) fulfillment is the capability to perform bare metal fulfillment. Bare metal
+fulfillment refers to the process through which a tenant acquires bare metal hosts and configures their networking.
+Afterwards, the tenant will be able to perform selected operations on the host: inventory information, power control, and
+serial console access. Any further operations - for example, image-based provisioning - are outside the scope of the O-SAC
+solution and must be integrated by the cloud provider.
```

Comment:
In our "virtual data center" model, we've discussed how one potential solution to provisioning would be to create virtual machine with your favorite provisioning tools and then attach to the same network as your bare metal hosts. Maybe we don't need to call that out here as an option, but what about:

```suggestion
solution and must be provided by the tenant or integrated by the cloud provider.
```